upgrades/upgrade-1.6-2.0.rst

```diff
@@ -0,0 +1,291 @@
+Upgrading from Sulu 1.6 to 2.0
+==============================
+
+We have a very extensive `UPGRADE.md file`_ in our repository, which contains almost every breaking change we made
+during the development of Sulu 2.0. However, this is a huge list and most projects won't be affected by the majority of
+these breaking changes. Therefore we are going through the most important changes in this guide, and by following these
+steps you can probably upgrade most of the projects, if you have not overridden some internal mechanisms.
+
+**1. Create a new** `sulu/skeleton`_ **project and import your projects** ``.git`` **folder**
+
+In our experience it makes more sense and will save you future headache if you start over with a new project, because
```

Comment:
`start over with a new project based on sulu/skeleton which is the preferred template for starting a  Sulu 2.0 project. The old sulu/sulu-minimal template and the deprecated sulu/sulu-standard template are not recommended for running a Sulu.20 project.`